docs/developer-notes.md

```diff
@@ -35,4 +35,6 @@ Die data-testid muss genau in dem Bereich liegen, in dem der Endanwender auch ta
 --
 
 ## StartView.page.ts
---
\ No newline at end of file
+--
+## FooterDataTable.page.ts
+TestId für die combobox(Einträge pro Seite), also der Pfeil zum öffnen der combobox im FooterDataTable fehlt für die Ergebnisliste Alle Schulen.
```

Comment:
Die Combobox sitzt in Vuetify, da kommen wir nur schlecht ran, um eine Test Id hinzuzufügen. Kannst du mit einem Selector arbeiten, der in etwa so funktioniert? `.v-data-table-footer__items-per-page .v-input`
Evtl. kann dir der Playwright Inspector auch sowas ausgeben.

getByRole('combobox') finde ich nicht eindeutig genug. Manchmal gibt es auf den Seiten mehrere Comboboxen.

Reply:
Dein Selektor funktioniert, danke